napari/conftest.py

```diff
@@ -37,15 +37,22 @@ def fetch(data_name):
 
 
 def pytest_addoption(parser):
-    """An option to show viewers during tests. (Hidden by default).
+    """Add napari specific command line options.
 
-    Showing viewers decreases test speed by about %18.  Note, due to the
-    placement of this conftest.py file, you must specify the napari folder (in
-    the pytest command) to use this flag.
+    --show-viewer
+        Show viewers during tests, they are hidden by default. Showing viewers
+        decreases test speed by about %18.
```

Comment:
```suggestion
        decreases test speed by about 18%.
```
? =) Also, maybe we can just round it to 20%? "decreases test speed by approximately 18.42927%" 😂

Reply:
That was not my comment, Talley's I think, but I changed it to say "by about 20%".